common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -125,6 +125,12 @@
     ],
     "sqlState" : "22003"
   },
+  "BOOLEAN_STATEMENT_EMPTY_ROWS" : {
```

Comment:
This name confuses slightly. We got not empty rows but one empty row. Let's correct it.

Reply:
How about `BOOLEAN_STATEMENT_WITH_AN_EMPTY_ROW` ?

Reply:
Or `BOOLEAN_STATEMENT_WITH_EMPTY_ROW`?

Reply:
SGTM